src/test/java/org/folio/dao/DataImportQueueDaoTest.java

```diff
@@ -54,118 +66,170 @@ public void setUp() {
   public void shouldGetQueueItem(TestContext context) {
     // given
     doAnswer((InvocationOnMock invocation) -> {
-      Promise<RowSet<Row>> promise = invocation.getArgument(2);
-      promise.complete(new LocalRowSet(5));
-      return null;
-    }).when(postgresClient)
-      .execute(anyString(), any(Tuple.class), ArgumentMatchers.<Promise<RowSet<Row>>>any());
+        Promise<RowSet<Row>> promise = invocation.getArgument(1);
```

Comment:
Two issues here:
- you were mocking `.execute` when the call was to `.select`
- `.select` has the `Promise` as the second argument, so we need `.getArgument(1)